Makefile

```diff
@@ -75,7 +75,7 @@ endif
 export ACK_GINKGO_DEPRECATIONS := 1.16.5
 export ACK_GINKGO_RC=true
 
-export PATH := $(REPO_ROOT)/$(TOOLS_BIN_DIR):$(PATH)
+export PATH := $(TOOLS_BIN_DIR):$(PATH)
```

Comment:
I think there may be a similar change required on line 202. Would you mind looking at that, since we're fixing up the Makefile?

Reply:
Good point. Fixed.